tests/common.py

```diff
@@ -43,6 +48,36 @@
 # fmt: on
 
 TEST_ADDRESS_N = tools.parse_path("m/44h/1h/0h/0/0")
+COMMON_FIXTURES_DIR = (
+    Path(__file__).parent.resolve().parent / "common" / "tests" / "fixtures"
+)
+
+
+def parametrize_using_common_fixtures(*paths):
+    fixtures = []
+    for path in paths:
+        fixtures.append(json.loads((COMMON_FIXTURES_DIR / path).read_text()))
+
+    tests = []
+    for fixture in fixtures:
+        for test in fixture["tests"]:
+            tests.append(
+                pytest.param(
+                    test["parameters"],
+                    test["result"],
+                    marks=pytest.mark.setup_client(
+                        passphrase=fixture["setup"]["passphrase"],
+                        mnemonic=fixture["setup"]["mnemonic"],
+                    ),
+                )
+            )
+
+    return pytest.mark.parametrize("parameters, result", tests)
+
+
+def _load_fixtures(path: Path):
```

Comment:
this seems unused?

Reply:
Fixed in a1c2b35699dfa759cc8ea045b0254f8984186133.